autowarefoundation/autoware
# Feature/ros visualizers

## Status
**DEVELOPMENT**

## Description
Added `rqt_graphprofiler` ROS profiling and visualization tool

## Related PRs
None


## Todos
See ros/src/util/packages/ros_visualizers/rqt_graphprofiler/README.md


## Steps to Test or Reproduce
See also ros/src/util/packages/ros_visualizers/rqt_graphprofiler/README.md:
`rosparam set enable_statistics true`

`rosrun rosprofiler rosprofiler`

`rosrun rosprofiler rosgrapher`

`rosrun rqt_graphprofiler rqt_graphprofiler`
